kit/azure/buildingblocks/aks-starterkit-composition/starterkit/main.tf

```diff
@@ -0,0 +1,113 @@
+resource "meshstack_project" "dev" {
+  metadata = {
+    name               = "${var.name_prefix}-dev"
+    owned_by_workspace = var.workspace_identifier
+  }
+  spec = {
+    display_name = "${var.name_prefix}-dev"
+    tags = {
+      "environment"          = ["dev"]
+      "LandingZoneClearance" = ["container-platform"]
+      "Schutzbedarf"         = ["public"]
+    }
+  }
+}
+
+resource "meshstack_project" "prod" {
+  metadata = {
+    name               = "${var.name_prefix}-prod"
+    owned_by_workspace = var.workspace_identifier
+  }
+  spec = {
+    display_name = "${var.name_prefix}-prod"
+    tags = {
+      "environment"          = ["prod"]
+      "LandingZoneClearance" = ["container-platform"]
+      "Schutzbedarf"         = ["public"]
+
+    }
+  }
+}
+
+resource "meshstack_tenant" "dev" {
+  metadata = {
+    owned_by_workspace  = var.workspace_identifier
+    owned_by_project    = meshstack_project.dev.metadata.name
+    platform_identifier = "aks.meshcloud-azure-dev"
+  }
+
+  spec = {
+    landing_zone_identifier = "likvid-aks-dev"
+  }
+}
+
+resource "meshstack_tenant" "prod" {
+  metadata = {
+    owned_by_workspace  = var.workspace_identifier
+    owned_by_project    = meshstack_project.prod.metadata.name
+    platform_identifier = "aks.meshcloud-azure-dev"
+  }
+
+  spec = {
+    landing_zone_identifier = "likvid-aks-prod"
+  }
+}
+
+resource "meshstack_building_block_v2" "repo" {
+  spec = {
+    building_block_definition_version_ref = {
+      uuid = "1357fbb1-dc85-40c2-944d-27c93b92c2c5"
+    }
+
+    display_name = "GitHub Repo ${var.repo_name}"
+    target_ref = {
+      kind       = "meshWorkspace"
+      identifier = var.workspace_identifier
+    }
+
+    inputs = {
+      repo_name = {
+        value_string = var.repo_name
+
+      }
+    }
+  }
+}
+
```

Comment:
t: I'd recommend adding a wait resource before provisioning the github connector building blocks because the bb and tenant creation will return immediately and it takes a while until github repo and aks namespace are ready.

Reply:
added time_sleep 45s